components/helpers/interfaces.tsx

```diff
@@ -59,4 +59,6 @@ export interface ContestFilterProps {
   setOpen: React.Dispatch<React.SetStateAction<boolean>>
   statusFilter: IFilter
   setStatusFilter: React.Dispatch<React.SetStateAction<IFilter>>
+  setSearchData?: React.Dispatch<React.SetStateAction<any>>
+  setIsSearching?: React.Dispatch<React.SetStateAction<boolean>>
```

Comment:
props should be defined on the same file as the component

Reply:
fixed